src/components/DataView/DataViewTHead.tsx

```diff
@@ -243,7 +247,7 @@ function DataViewTHead(props: DataViewTHeadProps) {
 					<StyledTh key="_actions" className={`
 						paddingRight
 						${ !props?.bulkActions?.length ? "paddingLeft" : "" }
-					`}>
+					`}> <span className="columnHeader">Actions</span>

```

Comment:
2 observations here... 1. This should be on the next line to improve readability and 2. The line below already does this. The only thing that needed to change was that it should always render instead of only when the props.onColumnsChange is false. If you look at the DataViewLocales.json file and look for the "actions" property you'll see it already says "Actions" so the text will appear twice.